its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -937,28 +951,73 @@ public void incrementalPrAnalysis_NoCache() throws IOException {
 
   @Test
   public void incrementalPrAnalysis_ProducesUnchangedFiles() throws IOException {
-    // ToDo: Compute hashes of files and store them to protobuf data
-    // ToDo: Populate server cache for "base-branch". File3 should get wrong hash. Might need change of license edition on Orchestrator
     String projectKey = "incremental-pr-analysis";
+    String baseBranch = "master";
+
     Path projectDir = TestUtils.projectDir(temp, "IncrementalPRAnalysis");
-    File expectedUnchangedFiles = new File(projectDir.resolve(".sonarqube\\conf\\UnchangedFiles.txt").toString());
+    // This is a temporary solution for uploading a valid cache.
+    // The file was generated from https://github.com/SonarSource/sonar-dotnet/commit/28c1224aca62968fb52b0332d6f6b7ef3da11f2b commit.
+    // In order to regenerate this file:
+    // - Update `FileStatusCacheSensor` to trim the paths correctly and save only the relative paths.
```

Comment:
Cannot this be done in the prototype branch, so we can just "execute" the step? It's not clear to me what action should I take there.

Reply:
I would not invest time in this. We plan to replace it soon.